packages/flutter/lib/src/services/keyboard_inserted_content.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// A class representing rich content (such as a PNG image) inserted via the
+/// system input method.
+///
+/// The following data is represented in this class:
+///  - MIME Type
+///  - Bytes
+///  - URI
+@immutable
+class KeyboardInsertedContent {
```

Comment:
Nit: Maybe KeyboardInsertionContent or KeyboardMediaContent to avoid the past tense "Inserted"?  Just a thought as I read through this, I think it's also fine as-is, up to you.

Reply:
I think inserted content is fine to be honest, considering that the handler is also called `onContentInserted`.

Reply:
Good call, we should match the handler :+1: